packages/next-on-pages/src/buildApplication/processVercelFunctions/invalidFunctions.ts

```diff
@@ -71,6 +76,69 @@ async function tryToFixNotFoundRoute(
 	}
 }
 
+/**
+ * Tries to fix potential unnecessary and invalid i18n functions from the Vercel build output.
+ *
+ * This is a workaround for Vercel creating invalid Node.js i18n functions in the build output, and
+ * is achieved by combing through the Vercel build output config to find i18n keys that match the
+ * invalid functions.
+ *
+ * @param collectedFunctions Collected functions from the Vercel build output.
+ * @param opts Options for processing Vercel functions.
+ */
+async function tryToFixI18nFunctions(
+	{ edgeFunctions, invalidFunctions, ignoredFunctions }: CollectedFunctions,
+	{
+		vercelConfig,
+		functionsDir,
+	}: Pick<ProcessVercelFunctionsOpts, 'functionsDir' | 'vercelConfig'>,
+): Promise<void> {
+	if (!invalidFunctions.size || !vercelConfig.routes?.length) {
+		return;
+	}
+
+	const foundI18nKeys = vercelConfig.routes.reduce((acc, route) => {
+		if ('handle' in route) return acc;
+
+		// Matches the format used in certain source route entries in the build output config.
+		// e.g. "src": "/(?<nextLocale>default|en|ja)(/.*|$)"
+		/\(\??<nextLocale>([^)]+)\)/
```

Comment:
`\??` indicates that `?` is optional, is that correct?

Reply:
oops!